### ISSUE TITLE
Convert sg:tags to mz:kind, mz:kind_detail

### ISSUE THREAD

**Issue by nvkelso** [whosonfirst-data/whosonfirst-data#990]

**Comment by nvkelso** [comment]
@nvkelso to add mapping file here, maybe that's leftover from last year's MZ category work?

**Comment by thisisaaronland** [comment]
@dphiffer - did this get done already?

**Comment by dphiffer** [comment]
No, this has not been done. `mz:category` is [a thing](https://github.com/whosonfirst/whosonfirst-categories/tree/master/mz), but `mz:kind` is not. And we still need to resolve concordances from SimpleGeo.

**Comment by nvkelso** [comment]
Kind is often the far right side of category, so this can all get done together.
